client/packages/invoices/src/InboundShipment/DetailView/modals/InboundLineEdit/utils.tsx

```diff
@@ -85,7 +85,21 @@ export const itemVariantColumn = (
   label: 'label.item-variant',
   width: 150,
   Cell: InboundLineItemVariantInputCell,
-  setter: updateDraftLine,
+  setter: patch => {
+    const { packSize, itemVariant } = patch;
+
+    if (itemVariant) {
+      const packaging = itemVariant.packagingVariants.find(
+        p => p.packSize === packSize
+      );
+      updateDraftLine({
+        ...patch,
+        volumePerPack: (packaging?.volumePerUnit ?? 0) * 1000 * (packSize ?? 1),
```

Comment:
`/ 1000` right? 0.1L should be like 0.0001m3 or something?

<img width="543" height="136" alt="Screenshot 2025-08-01 at 9 25 44 AM" src="https://github.com/user-attachments/assets/83b74e57-1284-4d11-ae6d-003f00d0f917" />
<img width="703" height="122" alt="Screenshot 2025-08-01 at 9 25 37 AM" src="https://github.com/user-attachments/assets/f555de19-2b54-4129-bb5f-c64a836d8655" />

Reply:
Oh yes~